setups/saivs_client_server.json

```diff
@@ -0,0 +1,34 @@
+{"NPU": [
```

Comment:
Please provide some document with the format description and purpose of each field.